agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -3697,6 +3697,13 @@ bool slave_thread::slave_fsm(bool &call_slave_select)
             return false;
         }
 
+        // All attributes which are not explicitely set below are set to
+        // default by the TLV factory, see WSC_Attributes.yml
+        if (!autoconfig_wsc_add_m1()) {
+            LOG(ERROR) << "Failed adding WSC M1 TLV";
+            return false;
+        }
```

Comment:
I don't see the need for this change. The controller now supports any order of these two TLVs, so we don't need to match the order of the golden unit.

Reply:
ok